app/swagger/schemas/intent_to_file.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module Swagger
+  module Schemas
+    class IntentToFile
+      include Swagger::Blocks
+
+      swagger_schema :IntentToFileBase do
+        key :required, %i[
+          id
+          creation_date
+          expiration_date
+          participant_id
+          source
+          status
+          type
+        ]
+        property :id, type: :string
+        property :creation_date, type: :string
+        property :expiration_date, type: :string
+        property :participant_id, type: :integer
+        property :source, type: :string
```

Comment:
You gave an example value to `status` & `type` - why not give examples to the rest?  i.e. [`id`, `creation_date`, `expiration_date`, `participant_id`].